types/spectest/tests/consensusdata/contributions_encoding.go

```diff
@@ -1,6 +0,0 @@
-package consensusdata
-
-// ContributionsEncoding tests encoding and decoding contributions
-func ContributionsEncoding() *SpecTest {
-	panic("implement")
```

Comment:
why is this deleted?

Reply:
It's already inside `encoding.go` (duplicate)